radio.k0swe.Kel_Agent.yml

```diff
@@ -0,0 +1,45 @@
+app-id: radio.k0swe.Kel_Agent
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+build-options:
+  append-path: /usr/lib/sdk/golang/bin
+  env:
+    - 'LDFLAGS='
+command: kel-agent
+finish-args:
+  - --share=network
+modules:
+  - name: kel-agent
+    sources:
+      - type: git
+        url: https://github.com/gorilla/websocket.git
+        tag: v1.4.2
+        commit: e8629af678b7fe13f35dff5e197de93b4148a909
```

Comment:
tag and commit are mismatched.

Reply:
Woops, fixed in https://github.com/flathub/flathub/pull/2360/commits/394522cc87910f04feaabfc21455f5602b4ad520